content/guides/end-to-end-testing/amazon-cognito-authentication.md

```diff
@@ -7,22 +7,23 @@ e2eSpecific: true
 
 ## <Icon name="graduation-cap"></Icon> What you'll learn
 
-- Programmatically authenticate with
-  [Amazon Cognito](https://aws.amazon.com/cognito) via a custom Cypress command
-- Adapting your [Amazon Cognito](https://aws.amazon.com/cognito) application for
-  programmatic authentication during testing
+- Authenticate with [`cy.origin()`](/api/commands/origin) or programmatically
+  (legacy) to [Amazon Cognito](https://aws.amazon.com/cognito) via a custom
```

Comment:
```suggestion
- Login to [Amazon Cognito](https://aws.amazon.com/cognito) through the UI with [`cy.origin()`](/api/commands/origin)
- Programmatically authenticate with [Amazon Cognito](https://aws.amazon.com/cognito) via a custom
```

Reply:
Not sure if we had a stance, but programmatic login is a valid flow and I don't necessarily fee like we should label it legacy. I think it would just depend on what makes the most sense for a user's test suite and CI setup.

Reply:
I think thats fair. I feel like we would want to encourage the use of `origin` over programmatic due to how involved programmatic can be, but it is for sure a valid flow and probably something we don't have to call legacy

Reply:
addressed in e14bf7170